metis-dereference/metis-dereference-rest/src/main/java/eu/europeana/metis/dereference/rest/config/SwaggerConfig.java

```diff
@@ -1,46 +1,30 @@
 package eu.europeana.metis.dereference.rest.config;
 
-import java.util.Collections;
-import net.sf.saxon.s9api.streams.Predicates;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.service.ApiInfo;
-import springfox.documentation.service.Contact;
-import springfox.documentation.spi.DocumentationType;
-import springfox.documentation.spring.web.plugins.Docket;
-import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 /**
  * Config for Swagger documentation
  */
 @Configuration
-@EnableSwagger2
 public class SwaggerConfig {
```

Comment:
Since Swagger will not be used, perhaps would be best to rename this class name to `OpenAPIConfig`

Reply:
Actually not.
Swagger is still used(check at the import packages above)
It is just the specification that changes to the newer model.
So I would keep it as such